.circleci/config.yml

```diff
@@ -240,8 +240,10 @@ workflows:
           container-image-name-updates: 'container=donate,tag=regression-${CIRCLE_SHA1}'
           post-steps:
             - slack/notify:
+                channel: deployments, regression-alarms
                 event: fail
                 template: basic_fail_1
+                mentions: @here
```

Comment:
ah actually thought I'd deleted this line. But maybe worth leaving it in temporarily to see what it does.